lizard_efcis/migrations/0073_auto_20150909_1339.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('lizard_efcis', '0072_auto_20150908_1659'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='wns',
+            name='wns_oms_ecolims',
+            field=models.CharField(verbose_name='omschrijving', max_length=255, null=True, editable=False, blank=True),
+            preserve_default=True,
+        ),
+        migrations.AlterField(
```

Comment:
Deze is raar, want er is niets gewijzigd. Of was er nog een typo uit een vorige commit ofzo?

Even voor de zekerheid :-)